fix: reworking the user-guide sections about IDEs (#2399)
codenvy/che-docs: modules/end-user-guide/pages/url-parameter-for-the-workspace-ide.adoc

:_content-type: CONCEPT
:navtitle: URL parameter for the workspace IDE
:description: URL parameter for the workspace IDE

> :keywords: url, urls, parameter, parameters, starting, start, workspace, workspaces, url-parameter, url-parameters, multiple-url-parameters, combine-url-parameters, several-url-parameters, add-url-parameters, concatenate, concatenate-multiple-URL-parameters, concatenate-URL-parameters, concatenating, concatenating-multiple-URL-parameters, concatenating-URL-parameters, concatenation, start-duplicate-workspaces, start-duplicated-workspaces, duplicate-workspace, duplicate-workspaces, duplicated-workspace, duplicated-workspaces, how-to-start-a-duplicate, how-to-start-duplicate, devfile, override-devfile-file-name, override-file-name, file-name, devfile-name, different-devfile-name, file-path, devfile-path, devfile-file-path, devfile-path-parameter, devfile-file-path-parameterdevfile, file-path, devfile-path, devfile-file-path, devfile-path-parameter, devfile-file-path-parameter

Reviewer: These super-long lists of keywords are not helpful. Including so many things dilutes the results. Please, remove the generic keywords (url, starting, concatenating, file name, file path, etc. etc. - these are just a few examples).

Author: Thanks for spotting this--leftovers from before this PR.
I've removed these.
Done.

// :page-aliases:

[id="url-parameter-for-the-workspace-ide_{context}"]

> 

Reviewer: ```suggestion
```

= URL parameter for the workspace IDE

> Che-Theia is the default integrated development environment (IDE) that a workspace loads with if no IDE is specified as a URL parameter as part of the URL for starting a new workspace.

Reviewer 2: ```suggestion
If the URL for starting a new workspace doesn't contain a URL parameter specifying the integrated development environment (IDE), the workspace loads with the default IDE: Che Theia.
```
